src/WebJobs.Script/Binding/FunctionBinding.cs

```diff
@@ -103,9 +103,13 @@ internal static Collection<FunctionBinding> GetBindings(ScriptHostConfiguration
                             {
                                 throw new InvalidOperationException("Http binding can only be used for output.");
                             }
+                            // TODO: Why is this here?
                             name = name ?? "res";
                             bindings.Add(new HttpBinding(config, name, FileAccess.Write, bindingMetadata.IsTrigger));
                             break;
+                        case BindingType.HttpTrigger:
```

Comment:
Just looking at this code, unclear what the difference between http and httpTrigger is. http is **always** a trigger, no?